CHANGELOG.md

```diff
@@ -6,7 +6,74 @@ All notable changes to this project will be documented in this file. The format
+* Added `siri:VehicleRef`to `ServiceGroup`. [#214](https://github.com/VDVde/OJP/pull/214)
+* Added `AllowedSystemId`s to `PlaceRefStructure`, specifiying the systems to be queried by `OJPPlaceInformationRequest`. [#204](https://github.com/VDVde/OJP/pull/204)
+* Added `PublicCode` of the journey to `DatedJourney` and `ContinuousService`. [#201](https://github.com/VDVde/OJP/pull/201)
+
+
+### Changed
+
+* Renamed elements in `ExchangePointsResponseGroup`. [#295](https://github.com/VDVde/OJP/pull/295)
+* Removed obsolete code in *OJPRequestSupport.xsd*. [#294](https://github.com/VDVde/OJP/pull/294)
+* `WaitDuration`: changed to standard order of attributes. [#286](https://github.com/VDVde/OJP/pull/286)
+* Renamed \*`TripLeg`\* to \*`Leg`\*. [#230](https://github.com/VDVde/OJP/pull/230) 
+* Replaced `EntitlementProductRef`s with a `EntitlementProductListStructure`, allowing for data like `EntitlementProductName` or `ValidityPeriod` for each `EntitlementProduct` in the list. [#229](https://github.com/VDVde/OJP/pull/229)
+* Redefined `VatRate` as a percentage instead of an enumeration. [#228](https://github.com/VDVde/OJP/pull/228)
+* Replaced `coord` with `location` in `PlaceTypeEnumeration`. [#218](https://github.com/VDVde/OJP/pull/218)
+* Renamed `PointOfInterestCode` and `AddressCode` to `PublicCode`, `PointOfInterestName` and `AddressName` to `Name`. [#217](https://github.com/VDVde/OJP/pull/217)
+* Corrected `schemaLocation` for *siri_reference-v2.0.xsd*. [#211](https://github.com/VDVde/OJP/pull/211)
+* Renamed `ErrorMessage` to `Problem`, `Code` to `Type`, `Text` to `Title` and added `Details` and `LogData`; added dedicated \*`ProblemStructure`s for each request, enumerating the possible types, some of which were previously only defined in the documentation, some not defined at all; added the `Problem` element to the \*`ResultStructure`s. Renamed some problem types, e.g., `FARES_`\* to `FARE_`\*, `MULTIPOINTTRIP_TOOMANYPOINTS` to `TRIP_MULTIPOINT_TOOMANYPOINTS`. [#203](https://github.com/VDVde/OJP/pull/203)
+* Bug fix: renamed `MinimumBookingPeriod.` to `MinimumBookingPeriod`. [#163](https://github.com/VDVde/OJP/pull/163)
+
+
+### Removed
+
+
+### Internal Use
+
+* New OJP.spp (XML Spy project file). [#297](https://github.com/VDVde/OJP/pull/297)
+* Changed location of automatically generated documentation. [#279](https://github.com/VDVde/OJP/pull/279)
+* Migrated from *Travis* to *GitHub Actions*. [#274](https://github.com/VDVde/OJP/pull/274)
+* Renamed `FareProductCodeType` to `FareProductIdType`. [#267](https://github.com/VDVde/OJP/pull/267)
+* Added permalinks to headers in gnerated html documentation. [#246](https://github.com/VDVde/OJP/pull/246)
+* Improved *title* attribute and cardinality in generated HTML documentation. [#233](https://github.com/VDVde/OJP/pull/233)
+
+
+
+
+## [1.1] - Not released yet (will be released as part of 2.0)
```

Comment:
The following updates should  be added to V1.1:

Added
* Added new files *README.md* and this *CHANGELOG.md*. [#130](https://github.com/VDVde/OJP/pull/130)

Removed
* Removed annotations in `choice` constructs. [#157](https://github.com/VDVde/OJP/pull/157)

Reply:
First item done (#130), still TODO: add line for #157.

Reply:
157 DONE as well.